components/Button/TabButton.tsx

```diff
@@ -1,25 +1,32 @@
 import React from 'react';
 import styled from 'styled-components';
 import Button from './Button';
+import calculatorIcon from '../../assets/svg/futures/calculator-icon.svg';
 
 export type TabButtonProps = {
 	title: string;
 	detail?: string;
 	badge?: number;
+	icon?: boolean;
 	active?: boolean;
 	onClick?: React.MouseEventHandler<HTMLButtonElement> | undefined;
 	disabled?: boolean;
 	noOutline?: boolean;
 };
 
-const TabButton: React.FC<TabButtonProps> = ({ title, detail, badge, active, ...props }) => {
+const TabButton: React.FC<TabButtonProps> = ({ title, detail, badge, active, icon, ...props }) => {
 	return (
 		<StyledButton {...props}>
 			<div>
 				<p className="title">{title}</p>
 				{detail && <p className="detail">{detail}</p>}
 			</div>
 			{!!badge && <div className="badge">{badge}</div>}
+			{icon ? (
+				<img className="icon" src={`${calculatorIcon}`} height={'15px'} width={'auto'} />
```

Comment:
I see what you're saying about wanting another icon in the future. The issue with this implementation is that we're hard-coding the calculator icon as a boolean instead of specifying the icon we would like to display.